frontend/src/API.js

```diff
@@ -1,19 +1,22 @@
 export const APIGET = async (url) => {
```

Comment:
If this file is about our API, maybe don't export API_URL and just use it here, in both methods? Code outside using this functions doesn't need to care about the URL.